python/cudf/cudf/core/mixins/reductions.pyi

```diff
@@ -0,0 +1,70 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
```

Comment:
Does this file exist to make mypy happy?

Reply:
Yes, exactly.